aws/components/datafeed/setup.ftl

```diff
@@ -238,14 +238,23 @@
                                                 cmkKeyId,
                                                 streamBackupLoggingConfiguration )]
 
+        [#-- Establish bucket prefixes --]
+        [#local streamS3DestinationPrefix = solution.Bucket.Prefix]
+        [#local streamS3DestinationErrorPrefix = solution.Bucket.ErrorPrefix]
```

Comment:
Should also use these at 248/249 if the intention was to show sharing

Reply:
Corrected in ff090c0